### DISCUSSION TITLE
Not able to view AR on Android devices where size of glb big

### DISCUSSION THREAD

**Discussion by NitinJ88** [google/model-viewer#3016] [Q&A]
### Description
When I try to launch AR viewer in my application for big size glb  ( >=3MB) then AR viewer does not pop-up on my Android device.
Looks like AR is loading in background and till that time browser time out the request.
I am getting following error on console:
"DOMException: Failed to execute 'requestSession' on 'XRSystem': The requested session requires user activation."

![ModelViewerError](https://user-images.githubusercontent.com/86343802/144447232-f9f8b7a5-4221-4f6d-87e6-06fce97b3486.JPG)

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: v1.2.1

### Browser Affected

- [*] Chrome, version: 96.0.4664.45
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [* ] Android 10
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ *] WebXR
- [ ] SceneViewer
- [ ] QuickLoo

**Comment by elalish** [comment]
Are you  trying to bypass our AR button? As the error says, XR requires user activation, so if it's not tied to a user click, the browser will block you, hence our button.
